scripts/abins/sdata.py

```diff
@@ -152,6 +183,29 @@ def get_empty(cls: SD, *,
     def get_frequencies(self) -> np.ndarray:
         return self._frequencies.copy()
 
+    def get_q_bins(self) -> np.ndarray:
+        if self._q_bins is None:
+            return None
+        else:
+            return self._q_bins.copy()
+
+    def get_q_bin_centres(self) -> np.ndarray:
+        q_bins = self.get_q_bins()
+        if q_bins is None:
+            raise ValueError("No q_bins on this SData")
+        else:
+            return (q_bins[1:] + q_bins[:-1]) / 2.
+
+    def set_q_bins(self, q_bins: np.ndarray) -> None:
+        """Update q-bins stored on SData
+
+        Args:
+            q_bins: 1-D set of q-point bin edges surrounding rows in S data
```

Comment:
can we have an example? It might make it easier to understand

Reply:
Would
```
(e.g. for q-bins=[1., 2., 3.], S arrays will have two rows
corresponding to q=1-2 Å^-1, q=2-3 Å^-1; typically these
are evaluated at the bin centres 1.5, 2.5 Å^-1.)
```
do the trick?

Reply:
swap the dash for a to and yes (since we are using dash for minus in the same statement)